bazel/external/quiche.BUILD

```diff
@@ -485,6 +485,40 @@ envoy_cc_library(
     ],
 )
 
+envoy_cc_library(
+    name = "quic_core_crypto_proof_source_interface",
```

Comment:
nit: Please keep the libraries name in sync with their internal names. Same for quic_core_crypto_proof_verifier_interface.

Reply:
Switched to quic_core_crypto_crypto_handshake_lib after sync'ing https://github.com/envoyproxy/envoy/pull/7340.